src/Misc/layoutbin/en-US/strings.json

```diff
@@ -157,6 +157,8 @@
     "--auth                 Authentication type. Valid options are PAT (Personal Access Token),",
     "                       Negotiate (Kerberos or NTLM), Integrated (Windows default credentials) and",
     "                       ALT (Alternate Credentials)",
+    "--work                 Agent work folder, default to _work folder under agent root,",
```

Comment:
I'm not sure what a good description would be. Perhaps:
```
Work directory where job data is stored. Defaults to _work under the root of the agent directory.
```

..this tells the user the purpose for the work directory (where job data is stored). Verbiage about relative paths may be overcomplicating it. The default is a relative path. If they aren't accepting the default, then the customer is probably going to specify a rooted path anyway. If not accepting the default, the chosen directory is probably at or near the root of a drive. Counting how many ".." are needed is not as natural as just specifying the absolute path for a folder at or near the root of a drive.